types/fancade-editor-script/tsconfig.json

```diff
@@ -0,0 +1,23 @@
+{
+    "compilerOptions": {
+        "module": "commonjs",
+        "lib": [
+            "ES5"
```

Comment:
This should match the expected config; see the bot warning and the README.

Reply:
I made the package `es6` again, so it should match the template now.